functions/common.php

```diff
@@ -168,3 +167,18 @@ function gnt_force_redirect($url) {
         </script><?php
     }
 }
+
+/**
+ * Hide Row if setting is disabled
+ * @param  string $setting The setting to check
+ * @echo string 'display:none;' if setting is disabled
```

Comment:
If the function does not return anything then please add a `@return null` not `@echo string 'display:none;' if setting is disabled`